components/governance/org.wso2.carbon.governance.api/src/main/java/org/wso2/carbon/governance/api/util/GovernanceUtils.java

```diff
@@ -64,17 +64,17 @@
 import org.wso2.carbon.utils.multitenancy.MultitenantConstants;
 import org.wso2.carbon.utils.multitenancy.MultitenantUtils;
 
+import java.io.StringReader;
+import java.util.*;
```

Comment:
Fixed with the following PR - https://github.com/wso2/carbon-governance/pull/254